pystac/common_metadata.py

```diff
@@ -23,17 +25,17 @@ class CommonMetadata:
         properties : Dictionary of attributes that is the Item's properties
```

Comment:
How is `properties` an `Arg` here?

Reply:
>IMO a fix doesn't belong in this PR -- we should open another issue, or find an existing documentation issue and link this problem in.

Yep, that's fine. I'm just not seeing a `properties` attribute or an argument/parameter named `properties` for `__init__()`, so not sure why it's in the docstring at all.

Reply:
I thinks it's referencing the `object` attribute -- for sure it's not right.